Profiles - ZIB 2017/zib-Wound.xml

```diff
@@ -0,0 +1,129 @@
+      </type>
+    </element>
+    <element id="Condition.extension:DateOfLastDressingChange.valueDate:valueDate">
+      <path value="Condition.extension.valueDate" />
+      <sliceName value="valueDate" />
+      <mapping>
+        <identity value="hcim-Wound-v3.2-2017EN" />
+        <map value="NL-CM:19.2.3" />
+      </mapping>
+    </element>
+    <element id="Condition.code">
+      <path value="Condition.code" />
+      <short value="WoundType" />
+      <definition value="Description of the type of wound." />
+      <alias value="WondSoort" />
+      <binding>
+        <strength value="extensible" />
+        <description value="WondSoortCodelijst" />
+        <valueSetReference>
+          <reference value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.19.2.4--20171231000000" />
+          <display value="ProbleemNaamCodelijst" />
+        </valueSetReference>
+      </binding>
+      <mapping>
+        <identity value="hcim-Wound-v3.2-2017EN" />
+        <map value="NL-CM:19.2.5" />
+      </mapping>
+    </element>
+    <element id="Condition.bodySite">
+      <path value="Condition.bodySite" />
+      <binding>
+        <strength value="extensible" />
+        <description value="WondAnatomischeLocatieCodelijst" />
+        <valueSetReference>
+          <reference value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.19.2.6--20171231000000" />
+          <display value="ProbleemAnatomischeLocatieCodelijst" />
+        </valueSetReference>
+      </binding>
+      <mapping>
+        <identity value="hcim-Wound-v3.2-2017EN" />
+        <map value="NL-CM:19.2.18" />
+      </mapping>
+    </element>
+    <element id="Condition.bodySite.extension:Laterality">
+      <path value="Condition.bodySite.extension" />
+      <sliceName value="Laterality" />
+    </element>
+    <element id="Condition.bodySite.extension:Laterality.valueCodeableConcept:valueCodeableConcept">
+      <path value="Condition.bodySite.extension.valueCodeableConcept" />
+      <sliceName value="valueCodeableConcept" />
+      <binding>
+        <strength value="extensible" />
+        <description value="WondLateraliteitCodelijst" />
+        <valueSetReference>
+          <reference value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.19.2.8--20171231000000" />
+          <display value="ProblemLateralityCodelist" />
+        </valueSetReference>
+      </binding>
+      <mapping>
+        <identity value="hcim-Wound-v3.2-2017EN" />
+        <map value="NL-CM:19.2.18" />
+      </mapping>
+    </element>
+    <element id="Condition.onsetDateTime:onsetDateTime">
+      <path value="Condition.onsetDateTime" />
+      <sliceName value="onsetDateTime" />
+      <mapping>
+        <identity value="hcim-Wound-v3.2-2017EN" />
+        <map value="NL-CM:19.2.2" />
+      </mapping>
+    </element>
+    <element id="Condition.note">
+      <path value="Condition.note" />
+      <mapping>

```

Comment:
Maybe put the mapping on Condition.note.text?

Reply:
okay